entrypoint.py

```diff
@@ -69,7 +185,8 @@ def check_file_content(self, url, data):
             if column_name not in json_response_df['name'].values:
                 errors.append(f"File has unrecognised column(s): {column_name}")
             else:
-                custom_verification = self.validate_uploaded_data(series, json_response_df[json_response_df['name'] == column_name])
+                custom_verification = self.validate_uploaded_data(series, json_response_df[
+                    json_response_df['name'] == column_name])
                 if custom_verification:  # If there's an error
                     errors.append(f"File is not valid for column: {column_name}")
```

Comment:
If it is possible it would be helpful if within the message can be also included what datatype is expected, or similar clue to know what is causing the error.